Trial implementation release candidate

c-corman opened the pull request:
## 📑 Description
Preparation for trial-implementation phase

- [x] Removed the "Tilda" from the Issues page.
- [x] Removed the NDH dependency
- [x] Small corrections in Volume 1
- [x] Assigned IDs in a sequence for all open and closed issues
- [x] Modified the sushi-config and publication-request files to prepare them for trial-implementation

## ✅ Checks
- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed
- [x] I have selected a committee co-chair to review the PR

JohnMoehrke commented:
CI Build -- https://build.fhir.org/ig/IHE/QRPH.mADX/branches/trial-implementation_releaseCandidate/

JohnMoehrke reviewed (dismissed):
You MUST resolve the QA issues. errors = 4, warn = 98, info = 0, broken links = 0

see https://build.fhir.org/ig/IHE/QRPH.mADX/branches/trial-implementation_releaseCandidate/qa.html

JohnMoehrke commented:
I see you have indicated some canonical are examples... If they are examples, please use the `http://example.org/` canonical root so that they are seen as examples.

c-corman commented:
I have addressed all the issues mentioned above.
At the moment there are 4 functional validation errors that should be fixed in the validation logic, as stated in this GitHub issue: https://github.com/hapifhir/org.hl7.fhir.core/issues/1927.
In order to address the warnings triggered by the usage of an example code system for derived periods, with help from @costateixeira, we have now a local code system "ISO8601DerivedPeriods" added on the mADX profile. This local code system could be replaced with an official code system once it's available, for example https://jira.hl7.org/browse/FHIR-50730.

@JohnMoehrke Please let us know if the merge can be approved now. Thanks!
